internal/repository/postgresdb/organizationConfig.go

```diff
@@ -0,0 +1,123 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/jmoiron/sqlx"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/apperrors"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/dto"
+	"github.com/joshsoftware/peerly-backend/internal/repository"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/constants"
+	logger "github.com/sirupsen/logrus"
+)
+
+type OrganizationConfigStore struct {
+	BaseRepository
+	OrganizationConfigTable string
+}
+
+func NewOrganizationConfigRepo(db *sqlx.DB) repository.OrganizationConfigStorer {
+	return &OrganizationConfigStore{
+		BaseRepository: BaseRepository{db},
+		OrganizationConfigTable: constants.OrganizationConfigTable,
+	}
+}
+
+func (org *OrganizationConfigStore) CreateOrganizationConfig(ctx context.Context,tx repository.Transaction, orgConfigInfo dto.OrganizationConfig) (createdOrganization repository.OrganizationConfig, err error) {
+
+	queryExecutor := org.InitiateQueryExecutor(tx)
+
+	insertQuery, args, err := repository.Sq.
+	Insert(org.OrganizationConfigTable).Columns(constants.OrgConfigColumns...).
+	Values(		1,
+		orgConfigInfo.RewardMultiplier,
+		orgConfigInfo.RewardQuotaRenewalFrequency,
+		orgConfigInfo.Timezone,
+		orgConfigInfo.CreatedBy,
+		orgConfigInfo.UpdatedBy,).
+	Suffix("RETURNING id, reward_multiplier ,reward_quota_renewal_frequency, timezone, created_by, updated_by, created_at, updated_at").
+	ToSql()
+	if err != nil {
+		logger.Errorf("err in creating query: %v",err)
+		return repository.OrganizationConfig{}, apperrors.InternalServer
+	}
+	
+	err = queryExecutor.QueryRowx(insertQuery, args...).StructScan(&createdOrganization)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			logger.Errorf("err in creating orgconfig : %v ", err)
+			return repository.OrganizationConfig{}, apperrors.InternalServer
+		}
+	}
+	return
+}
+
+func (org *OrganizationConfigStore) UpdateOrganizationConfig(ctx context.Context, tx repository.Transaction, reqOrganization dto.OrganizationConfig) (updatedOrganization repository.OrganizationConfig, err error) {
+	queryExecutor := org.InitiateQueryExecutor(tx)
+	
+	updateBuilder := repository.Sq.Update(org.OrganizationConfigTable).
+	Where(sq.Eq{"id": 1}).
```

Comment:
create and use const as DefaultOrgID or similar.

Reply:
created constant in system.go file and used in updateOrganizationConfig and getOrganization repo function